mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
> but I've seen some other config files without it

Those are also wrong.

> I'm leaving it as it is to match the current config files.

That makes no sense. Some config files have this error, others do not, and you’re choosing to follow the objectively wrong behaviour.

The files that have this error should all be fixed.